tests/integration/long/test_failure_types.py

```diff
@@ -301,3 +302,75 @@ def test_user_function_failure(self):
             """
             DROP TABLE test3rf.d;
             """, consistency_level=ConsistencyLevel.ALL, expected_exception=None)
+
+
+class TimeoutTimerTest(unittest.TestCase):
```

Comment:
My only concern with adding that sort of coverage without a node pausing, is that it might not be deterministic. Especially with ccm we often see very quick response times. I'll see what I can maybe I can make something work